docs/api/check.md

```diff
@@ -9,3 +9,7 @@
 # Simple SRM Check
 
 ::: epstats.toolkit.SimpleSrmCheck
+
+# SumRatio Check
```

Comment:
Maybe consider also adding a specific page for SumRatioCheck, just like there is one for SRMCheck (or is it unnecessary since it's just a duplicate?)

Reply:
ohh right, what about deleting the `srm_check.md` and having just one file to keep it simple?

Reply:
Yep, that's also fine, I'd just keep it consistent